dali/pipeline/operators/crop/basic/coords.h

```diff
@@ -0,0 +1,109 @@
+// Copyright (c) 2018, NVIDIA CORPORATION. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef DALI_PIPELINE_OPERATORS_CROP_BASIC_COORDS_H_
+#define DALI_PIPELINE_OPERATORS_CROP_BASIC_COORDS_H_
+
+#include <array>
+#include <cstdint>
+#include <iostream>
+
+namespace dali {
+namespace basic_crop {
+
+namespace detail {
+
+template <typename T, size_t N>
+int64_t getOffsetImplLinear(const T& shape, std::array<int64_t, N> coords) {
+  int64_t result = coords[0];
+  for (size_t i = 1; i < N; i++) {
+    result *= shape[i];
+    result += coords[i];
+  }
+  return result;
+}
+
+}  // namespace detail
+
+template <int64_t... Ints>
+class dali_index_sequence {};
+
+template <typename T>
+struct index_to_array {};
+
+template <int64_t... Ints>
+struct index_to_array<dali_index_sequence<Ints...>> {
+  static constexpr std::array<int64_t, sizeof...(Ints)> array = {Ints...};
+  static constexpr size_t N = sizeof...(Ints);
```

Comment:
Providing unit test for this also will awesomely document the entity (how to use it, if it works etc...)

Reply:
I removed it, as this was dead code. Still, it just unpacked the template argument pack of ints to array.